### PR TITLE
[TG-TECH][chore] add command line interface

### PR DESCRIPTION
**Pull request by vreshch**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const colors = require('colors');
+const fs = require('fs');
+const program = require('commander');
+const exec = require('shelljs').exec;
+const path = require('path');
+
+/**
+* Init program version
+*/
+const packageJSON = JSON.parse(fs.readFileSync('package.json'));
```
Comment (disist) [inline index.js:12]: Ooo, maybe better to try `process.env.npm_package_version`

File: package.json [hunk 2]
```diff
@@ -17,19 +21,20 @@
...
   "engines": {
-    "node": ">=5"
-  },
-  "devDependencies": {
-    "npm-run-all": "^2.2.2"
+    "node": "5.x.x"
   },
   "dependencies": {
+    "commander": "^2.9.0",
+    "cross-env": "^2.0.1",
```
Comment (disist) [inline package.json]: I try to find `cross-env` usage, maybe it's not needed?

File: package.json [hunk 3]
```diff
@@ -17,19 +21,20 @@
...
+    "shelljs": "^0.7.0",
+    "colors": "1.1.2",
```
Comment (disist) [inline package.json]: I try to find `colors` usage, maybe it's not needed?

Comment (vreshch) [reply package.json]: Thx. miss that

File: index.js (hunk 1 above)
Comment (vreshch) [reply index.js:12]: Good idea, but seems that this approach will work only when you start with npm, but will not work any longer for global call.

File: package.json (hunk 3 above)
Comment (vreshch) [reply package.json]: -- removed. Thx.

File: package.json (hunk 2 above)
Comment (vreshch) [reply package.json]: -- removed. Thx.

**Review by kucherenko — APPROVED** [review]

**Pull request merged**